Makefile

```diff
@@ -561,12 +561,15 @@ dev-flavors: $(OVERRIDES_DIR)
 
 .PHONY: generate-flavors
 generate-flavors: $(FLAVOR_DIR)
-	go run ./packaging/flavorgen -f vip > $(FLAVOR_DIR)/cluster-template.yaml
-	go run ./packaging/flavorgen -f external-loadbalancer > $(FLAVOR_DIR)/cluster-template-external-loadbalancer.yaml
-	go run ./packaging/flavorgen -f cluster-class > $(FLAVOR_DIR)/clusterclass-template.yaml
-	go run ./packaging/flavorgen -f cluster-topology > $(FLAVOR_DIR)/cluster-template-topology.yaml
-	go run ./packaging/flavorgen -f ignition > $(FLAVOR_DIR)/cluster-template-ignition.yaml
-	go run ./packaging/flavorgen -f node-ipam > $(FLAVOR_DIR)/cluster-template-node-ipam.yaml
+	go run ./packaging/flavorgen --output-dir $(FLAVOR_DIR)
+
+.PHONY: verify-flavors
+verify-flavors: $(FLAVOR_DIR)
+	go run ./packaging/flavorgen --output-dir $(FLAVOR_DIR)
```

Comment:
```suggestion
verify-flavors: $(FLAVOR_DIR) generate-flavors
```

Reply:
Also: could we move this target to the other `verify-...` targets inside the makefile :-)

Reply:
Sure :-)